_includes/releases/v23.1/v23.1.0-alpha.8.md

```diff
@@ -0,0 +1,139 @@
+
+- Enabled the `changefeed.mux_rangefeed.enabled` [cluster setting](../v23.1/cluster-settings.html) for MuxRangeFeed clients to increase the efficiency when running against large scale workloads. [#97957][#97957]
+- The `server.oidc_authentication.claim_json_key` [cluster setting](../v23.1/cluster-settings.html) for DB Console SSO now accepts list-valued token claims. [#98522][#98522]
+- Added the `WITH` key_column option to override the message metadata key for [changefeeds](../v23.1/changefeed-examples.html). This changes the key hashed to determine Kafka partitions. It does not affect the output of `key_in_value` or the domain of the per-key ordering guarantee. [#98806][#98806]
+- Normalized CPU usage will now show on [Node Map](../v23.1/ui-cluster-overview-page.html#node-map-enterprise). [#98225][#98225]
+
+<h3 id="v23-1-0-alpha-8-sql-language-changes">SQL language changes</h3>
+
+- Fixed a bug where the check constraint on an `OID` type column panics in the legacy schema changer. [#98800][#98800]
+- Added a new `WITH REDACT` option to the following statements: [`SHOW CREATE`](../v23.1/show-create.html), [`SHOW CREATE TABLE`](../v23.1/show-create.html#show-the-create-table-statement-for-a-table), and [`SHOW CREATE VIEW`](../v23.1/show-create.html#show-the-create-view-statement-for-a-view)  which, when used, replaces constants and literals in the printed `CREATE` statement with the redacted marker, `‹×›`. [#98251][#98251]
+- Disallowed the [`RESTORE`](../v23.1/restore.html) of backups taken on a cluster version older than the minimum binary version the current cluster can interoperate with. This is described in an updated version of the policy outlined in ["Restoring Backups Across Versions"](../v23.1/restoring-backups-across-versions.html). [#98597][#98597]
+- Bulk [`COPY FROM`](../v23.1/copy-from.html) statements are now processed with a vectorized insert and can be anywhere from 0.5x to 5x faster. Typical hardware and schemas should see a 2x improvement. Vectorized inserts are only used for `COPY` statements and are not yet applied to regular inserts. Both the `vectorize` and `copy_fast_path_enabled` session variables can be used to disable this feature. [#98605][#98605]
+- Added support for the `REDACT` flag to the following variants of [`EXPLAIN`](../v23.1/explain.html): `EXPLAIN (OPT)`, `EXPLAIN (OPT, CATALOG)`, `EXPLAIN (OPT, MEMO)`, `EXPLAIN (OPT, TYPES)`, and `EXPLAIN (OPT, VERBOSE)`.  These `EXPLAIN` statements will have constants, literal values, parameter values, and any other user data redacted in the output. [#97549][#97549]
+- Added a `sql.pgwire.multiple_active_portals.enabled` setting. This setting is only for a `PREVIEWABLE` feature. With it set to `true`, all non-internal portals with read-only [`SELECT`](../v23.1/selection-queries.html) queries without sub/post queries can be paused and resumed in an interleaving manner, but are executed with a local plan. [#99052][#99052]
```

Comment:
```suggestion
- Added the setting `sql.pgwire.multiple_active_portals.enabled`. This setting is only for a `PREVIEWABLE` feature. With it set to `true`, all non-internal portals with read-only [`SELECT`](../v23.1/selection-queries.html) queries without sub/post queries can be paused and resumed in an interleaving manner, but are executed with a local plan. [#99052][#99052]
```

Reply:
@ZhouXing19 Could you please briefly explain what it means that 'This setting is only for a PREVIEWABLE feature.'?

@ashleykwtam I might merge as-is, for timing reasons, but come back and update if we can get more clarity in the thread.

Reply:
This setting is for a feature for preview only, meaning it may have vulnerability, and shouldn't be applied at a large scale of application and not be used in production. 
Actually, is it ok for us not to include it in the release note for now? In [a PR that is going to be merged soon](https://github.com/cockroachdb/cockroach/pull/99663) we're going to replace this cluster setting with a session variable.
cc @rafiss @dikshant

Reply:
just to clarify, i think "vulnerability" is the wrong word to use here - that usually means "security vulnerability"

it's in preview because it has unknown performance effects

the definition of preview feature is here: https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability.html#feature-availability-phases

Reply:
Should we remove this from release notes since it will be changed for the next alpha? @mikeCRL

Reply:
Thanks, all. Yes, @ashleykwtam please do remove this.

Reply:
removed from release notes